commandlineparser/src/Parser.cpp

```diff
@@ -1,29 +1,5 @@
 #include "Parser.h"
 
```

Comment:
So this was was my point of  private constructor  delegating the other paramter along with  shared pointer<Options> obj

Reply:
done .